src/PcapngUtils/PcapngUtils.csproj

```diff
@@ -5,20 +5,22 @@
     <RootNamespace>Haukcode.PcapngUtils</RootNamespace>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
-    <Version>0.999-debug</Version>
```

Comment:
We shouldn't change this version, it's used when building locally, so it should be version 0.999

Reply:
done